src/server/scripts/Kalimdor/TempleOfAhnQiraj/boss_cthun.cpp

```diff
@@ -342,10 +342,10 @@ class boss_eye_of_cthun : public CreatureScript
                         if (DarkGlareTickTimer <= diff)
                         {
                             //Set angle and cast
-                            if (ClockWise)
-                                me->SetOrientation(DarkGlareAngle + DarkGlareTick * float(M_PI) / 35);
-                            else
-                                me->SetOrientation(DarkGlareAngle - DarkGlareTick * float(M_PI) / 35);
+                            float angle = (ClockWise) ? DarkGlareAngle + DarkGlareTick * float(M_PI) / 35 : DarkGlareAngle - DarkGlareTick * float(M_PI) / 35;
```

Comment:
Removing the parenthesis as they are not needed and we rarely have them in the rest of the code
```suggestion
                            float angle = ClockWise ? DarkGlareAngle + DarkGlareTick * float(M_PI) / 35 : DarkGlareAngle - DarkGlareTick * float(M_PI) / 35;
```

Reply:
Updated